### PR TITLE
Setup local Supabase

### PR DESCRIPTION
**Pull request by tselmek**
# What this PR does
- Initialise and document local Supabase setup

### CODE REVIEW

**Review by damien-schneider — COMMENTED** [review]
Wow, very nice documentation, thank you so much !

File: supabase/config.toml
Comment (damien-schneider) [inline supabase/config.toml:1]: How did you get this config, is it an export from supabase studio ?

File: supabase/functions/.vscode/settings.json
Comment (damien-schneider) [inline supabase/functions/.vscode/settings.json:1]: What are these deno settings ? Is it something you took from the supabase doc ?

**Review by damien-schneider — COMMENTED** [review]
Thank you very much for this PR !
I just asked few question before merging to understand a little bit more because local dev with supabase is a little bit new for me :)

File: supabase/config.toml
Comment (tselmek) [reply supabase/config.toml:1]: Almost all config files were generated automatically by the `supabase init` initialisation command

File: supabase/functions/.vscode/settings.json
Comment (tselmek) [reply supabase/functions/.vscode/settings.json:1]: Same as other comment, it is automatically generated by the `supabase init` command on initialisation

Comment (damien-schneider) [reply supabase/functions/.vscode/settings.json:1]: Ok, perfect, thank you very much !

**Review by damien-schneider — APPROVED** [review]

**Pull request merged**